### ISSUE TITLE
Define the “decision making flow” that the system could apply to auto accept a pre-registration project

### ISSUE THREAD

**Issue by clairerye** [ebi-ait/hca-ebi-dev-team#214]
## user story
As a contributor with a pre-registered project that is unambiguously acceptable, I want to download a template spreadsheet immediately after providing some experiment information

## task
Define the “decision making flow” that the system could apply to auto accept a pre-registration project, otherwise wrangler review as planned.
This may be: (Human, open access, defined technology)

**Comment by lauraclarke** [comment]
A quick comment, it isn't just human, it is human, primary tissue, and deceased donors with limited privacy implications in most legal jurisdictions

**Comment by ami-day** [comment]
I am listing the requirements **I think** we have currently but I will need input from Laura and other wranglers. I am not the best person to do this though, as previously when I have asked these questions I have got mixed responses. So please feel free to edit!

**(1) Data Access**
- If the organism is answered as "Human" or "both" in the project registration form, then the contributor is required to select "fully open" for the project to be unambiguously accepted.
- If the organism is answered as "Mouse", we do not require any particular data access question but the contributor would most likely select "fully open".
_Laura, I read your comment. I am not sure how to address it here, because we do not ask about the sample type or health status in the project registration form._

**(2) Organism**
The organism must be "Human", "Mouse" or both of those.

**(3) Technology**
Currently, there is not a requirement (?) as long as it is a single-cell RNA sequencing method. If they select a technology that is in the drop-down list, we can unambiguously accept the project. If they only write a technology name in the "Other technologies" box, then a wrangler will need to review this is indeed a single-cell RNA sequencing method.

**Comment by lauraclarke** [comment]
@ami-day as we aren't going to definitely take cell line or humanised mouse projects, I am worried about any auto acceptance criteria which can't check the project doesn't use human immortalized cell likes or some complex gene editing strategy

**Comment by clairerye** [comment]
@lauraclarke So in practice, have we ever actually rejected a project that was cell line or weird mouse? If not, it feels like we could have very simple logic to cover most yes submissions, or more complicated logic for the odd edge case.

**Comment by lauraclarke** [comment]
Yes, we have said no though right now mostly for non-human or mouse but we would also say no to humanised mice and cancer cell lines and there are definitely published datasets which are cell lines or humanized mice that we initially prioritized and then had to stop working on because it became clear they didn't meet our criteria

If there are human or mouse experiments where there are criteria we would say no that we don't automatically collect, we can't automatically say yes to any experiment

**Comment by ami-day** [comment]
@lauraclarke @clairerye So does this mean we have 2 options:

1. Change the project registration from to include a question such as "Are your samples predominantly immortalised cell lines or genetically modified mouse? or "What sample type was used (e.g. primary tissue; organoid; cell line, iPSC, etc.). This will enable us to still automatically accept a project.

2. We don't implement an automatic acceptance feature, because we don't know the sample type.

Also tagging @gabsie since it would potentially involve changing the design.

**Comment by ami-day** [comment]
I would say the 1st option would be most effective based on my experience working on the 2nd Laurenti project. They were able to fill a spreadsheet before even contacting us based on their previous experience, and it made the process very quick.

**Comment by lauraclarke** [comment]
While giving people spreadsheets quickly is a great plan we should reserve caution on basing contributor expectations on the Laurenti lab, they are the only repeat submitter who has behaved that way
